exotica_examples/src/exotica_examples_py/target_marker.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+from interactive_markers.interactive_marker_server import *
+from visualization_msgs.msg import *
+from geometry_msgs.msg import Pose, Point, Quaternion
+import PyKDL as kdl
+import pyexotica as exo
+
```

Comment:
Add an `__all__ = [list_to_pose, TargetMarker]` here - so when people do `from exotica_examples_py import *` they do not get transient access to PyKDL etc.